drivers/storage/portworx/deployment.go

```diff
@@ -0,0 +1,670 @@
+package portworx
+
+import (
+	"fmt"
+	"path"
+	"strconv"
+	"strings"
+
+	"github.com/google/shlex"
+	corev1alpha1 "github.com/libopenstorage/operator/pkg/apis/core/v1alpha1"
+	"github.com/sirupsen/logrus"
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+const (
+	pxContainerName          = "portworx"
+	pxServiceAccount         = "px-account"
+	annotationIsDocker       = "portworx/is-docker"
```

Comment:
let's use `portworx.io` for all annotation prefixes. Use a prefix const and create all annotation keys from that.